style.css

```diff
@@ -121,3 +121,646 @@ a {
+  font-size: 13px;
+  line-height: 1rem;
+  color: #344563;
+  margin-left: 0.4rem;
+}
+
+.work-card .align ul li {
+  width: 88px;
+  height: 1rem;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 1rem;
+  color: #7a869a;
+  justify-content: space-between;
+  margin-left: 2rem;
+}
+
+.work-card p {
+  width: 100%;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-left: 0.5rem;
+}
+
+.tags {
+  width: 100%;
+}
+
+.tags .hash {
+  display: flex;
+  list-style: none;
+  padding-right: 4.5rem;
+  gap: 0.5rem;
+  margin-left: 0.2rem;
+}
+
+.tags .hash .HTML {
+  padding: 4px 12px;
+  width: 57px;
+  height: 1.5rem;
+  background: #ebebff;
+  border-radius: 0.5rem;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 1rem;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.tags .hash .CSS {
+  padding: 4px 12px;
+  width: 49px;
+  height: 1.5rem;
+  background: #ebebff;
+  border-radius: 8px;
+  order: 1;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 1rem;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.tags .hash .JavaScript {
+  padding: 4px 12px;
+  width: 92px;
+  height: 1.5rem;
+  background: #ebebff;
+  border-radius: 0.5rem;
+  order: 2;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 1rem;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.submit {
+  width: 100%;
+}
+
+.submit button {
+  box-sizing: border-box;
+  padding: 12px;
+  gap: 10px;
+  width: 126px;
+  height: 48px;
+  border: 1px solid #6070ff;
+  border-radius: 0.5rem;
+  background-color: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 25px;
+  display: grid;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  margin-right: 10rem;
+  margin-top: 0.6rem;
+}
+
+.resume button {
+  box-sizing: border-box;
+  padding: 12px;
+  margin-left: 1.5rem;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  background-color: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 1.5rem;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  margin-right: 10rem;
+  margin-top: 1.5rem;
+}
+
+.someter button {
+  width: 142px;
+  box-sizing: border-box;
+  height: 50px;
+  border: 5px solid #6070ff;
+  border-radius: 8px;
+  background-color: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 1.5rem;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  margin-right: 10rem;
+  margin-top: 1.5rem;
+  display: flex;
```

Comment:
- [ ] It is noticed that `display: flex` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.flex-row` or `.flex-column` and assigning it wherever possible as a class name in your `html` markup code.
```
.flex-row {
  display: flex;
}

.flex-column {
  display: flex;
  flex-direction: column;
}
```
- And kindly attend to any repetitions in your code 🙂.
- Kindly note that DRY is a requirement.